### DISCUSSION TITLE
NEO2 Layout and physical key assignments

### DISCUSSION THREAD

**Discussion by rumpelsepp** [wezterm/wezterm#1799] [Q&A] [answered]
hey, I am a user of the [neo2 keyboard layout](https://neo-layout.org/). After the last upgrade I am unable to use wezterm since all keybindings changed for me due to [this](https://wezfurlong.org/wezterm/config/keys.html#physical-vs-mapped-key-assignments). How can I revert to the mapped behaviour before 20220319-142410-0fcdea07 without maintaining every single keybinding in my config? Thanks! :)

**Comment by bew** [comment, marked as answer]
Looks like you're asking the same thing as https://github.com/wez/wezterm/issues/1788#issuecomment-1082450965
